src/index.html

```diff
@@ -8,7 +8,23 @@
   <title>Webpack Exercise</title>
 </head>
 
-<body class=" bg-slate-900 w-full h-full">
+<body
+  class="grid grid-rows-[3rem_auto_3rem] grid-cols-1 md:grid-cols-2 lg:grid-cols-3 p-6 bg-white w-full min-h-screen">
+
+  <header
+    class="flex justify-between items-center col-span-full row-span-1 bg-sky-700 text-white">
+    <nav class="flex justify-between items-center w-full">
+      <a class="block px-3" href="#">MyLgo</a>
+      <ul
+        class="flex justify-between items-center px-3 gap-6">
+        <li><a href="#">Spaceships <span>6</span></a></li>
+        <li><a href="#">Planets <span></span></a></li>
+        <li><a href="#">Races <span></span></a></li>
+      </ul>
+    </nav>
+  </header>
```

Comment:
Good job so far @Sboursen, But there are some changes that you can make to the header, I think that it's better if you remove the padding to cover the whole space.

Reply:
Hi @mwafrika 
Thank you for the feedback
I'm on it